corehq/motech/openmrs/openmrs_config.py

```diff
@@ -41,6 +49,21 @@ def get_value(self, case_trigger_info):
         return self.value
 
 
+class CasePropertyConcept(CaseProperty):
```

Comment:
Is this class a separate feature, or is it added for this import-from-openmrs feature? (Fine if this PR contains a couple different things, just trying to understand.)

Reply:
You're right @dannyroberts this is separate. I think I'll move it to a separate PR, not just to make this less confusing, but also because it would be useful to have it deployed sooner.

Reply:
Ah, okay cool. Makes total sense. Thanks!